.github/workflows/add-to-projects.yml

```
@@ -36,17 +36,15 @@ jobs:
         with:
           project-url: https://github.com/orgs/camunda/projects/92
           github-token: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
-          labeled: kind/task
-          label-operator: NOT
+          labeled: component/zeebe
       - id: add-to-zda
```

🔧 I know you didn't change it, but you would mind changing this typo? 😄 

```suggestion
      - id: add-to-zpa
```